src/nidmm/system_tests/test_system_nidmm.py

```diff
@@ -132,3 +132,10 @@ def test_get_dev_temp(device_info):
         temperature = session.get_dev_temp('')
         print(temperature)
         assert 20 <= temperature <= 50
+
+def test_writeonly_attribute(device_info):
+    with nidmm.Session(device_info['name']) as session:
+        try:
+            session.channel_count = 5
+        except nidmm.Error as e:
+            assert e.code == -1074135027
```

Comment:
I would also like to see a comment about what the error is. I think the description should work.

This way if the error code ever changes, there will at least be some information to help hunt for the new appropriate error code.